src/lines.jl

```diff
@@ -35,9 +60,17 @@ function graphline(g::AbstractGraph{T}, locs_x, locs_y, nodesize::Real, arrowlen
         endx = locs_x[j] + nodesize*cos(θ+π)
         endy = locs_y[j] + nodesize*sin(θ+π)
         lines[e_idx] = [(startx, starty), (endx, endy)]
-        arr1, arr2 = arrowcoords(θ, endx, endy, arrowlength, angleoffset)
-        arrows[e_idx] = [arr1, (endx, endy), arr2]
+		if chainGraph
```

Comment:
You seem to have used tabs here. For consistency with the rest of the project we should spaces instead.